main.py

```diff
@@ -10,6 +10,15 @@ async def root():
     return 'Hello'
 
 
-@app.get('/movies')
-async def get_movies() -> list[Movie]:
+@app.get('/movies', response_model=list[Movie])
```

Comment:
Why did you switch from a standard type annotation to this?

Reply:
`fastapi` doesn't use the type annotations for output validation

Reply:
Oh weird.
Do you want output validation though, isn't mypy enforcing the return type enough?

Reply:
mypy is enough to validate the output, but this also updates the schema and shows it in the API docs.

I'll use both.

Reply:
Oh wow I completely forgot this annoying behavior, that's shit